microsoft-edge/progressive-web-apps-chromium/how-to/origin-trials.md

```diff
@@ -8,37 +8,62 @@ ms.service: microsoft-edge
 ms.subservice: pwa
 ms.date: 10/20/2023
 ---
-# Experimental features and origin trials
+# Experimental features and origin trials for PWAs
 
-Some of the PWA features in Microsoft Edge are still experimental.  Experimental features can be used in either of two ways:
+<!-- todo: add scope extensions OT, per
```

Comment:
Add a section about scope extensions, in PWA OTs article.
@captainbrosset

The article section
https://review.learn.microsoft.com/en-us/microsoft-edge/progressive-web-apps-chromium/how-to/handle-urls?branch=pr-en-us-3288#handle-links-from-other-origins-by-using-scope-extensions
is about scope extensions (an OT), yet the PWA OTs article doesn't mention "scope" or "extension":
https://review.learn.microsoft.com/en-us/microsoft-edge/progressive-web-apps-chromium/how-to/origin-trials?branch=pr-en-us-3288

This OT is Active, at Chrome status site.
Added a hidden todo comment in both articles.

Reply:
I don't think we should document the active OTs on the PWA OTs article. The OT portal will have the up to date information already. If we try to duplicate that information on our own OT-related articles, we're bound to get out of date eventually.

Mentioning specific OTs on specific feature pages, such as the handle-urls article seems ok to me, but also bound to get out of date at some point. But I wouldn't want to duplicate this information any further.

Reply:
I will remove todo comment from both articles.  I don't recall my context except that it was .md POV in VS Code, maybe i should've checked rendered docs POV to remember what kind of OT content in which kinds of articles eg What's New PWAs.

Reply:
thx for checking on this comment, i didn't intend to recommend covering a specific pwa OT in the general article about PWA ot's.  @captainbrosset